Doctrine/Phpcr/SitemapDocumentProvider.php

```diff
@@ -42,7 +42,7 @@ public function load($sitemap)
     {
         // todo rewrite query when https://github.com/symfony-cmf/CoreBundle/issues/126 is ready
         $documentsCollection = $this->manager->createQuery(
-            'SELECT * FROM [nt:unstructured] WHERE (visible_for_sitemap = true)',
+            'SELECT * FROM [nt:unstructured] WHERE (visible_for_sitemap = true) ORDER By [jcr:path]',
```

Comment:
@dbu created a special function test for that. That test is a green test, but i inserted `DESC` and got the fail.